### PR TITLE
Adding a Node FAQ section 

### PR DESCRIPTION
**Pull request by dubrado**
Creating a section for Node FAQs, so we can populate common errors and fixes.

### CODE REVIEW

**Review by yulin-dong — CHANGES_REQUESTED** [review]
Please fix merge conflict first.

File: docs/nodes/build/run-avalanche-node-manually.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Download the AvalancheGo repository into your `$GOPATH`:
 cd $GOPATH
 mkdir -p src/github.com/ava-labs
 cd src/github.com/ava-labs
-git clone git@github.com:ava-labs/avalanchego.git
+git clone https://github.com/ava-labs/avalanchego
```
Comment (yulin-dong) [inline docs/nodes/build/run-avalanche-node-manually.md:111]: Git SSH is still much widely used than https.

File: docs/nodes/FAQ/Node-FAQ.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Frequently Asked Questions
```
Comment (yulin-dong) [inline docs/nodes/FAQ/Node-FAQ.md:1]: Please turn the PR to draft, if you are not ready to merge yet.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: sidebars.js [hunk 3]
```diff
@@ -288,6 +288,17 @@ const sidebars = {
...
       ]
+    },
+    {
+      type: 'category',
+      label: 'FAQ',
+      collapsed: false,
+      items: [
+        {
+          type: 'autogenerated',
+          dirName: 'nodes/FAQ'
```
Comment (yulin-dong) [inline sidebars.js:299]: FAQ is not a directory, it is a file only.

Comment (dubrado) [reply sidebars.js:299]: I left it open to possibly add more so if the FAQ grows we can add future files, but open to whatever works best in the short term.

Comment (dubrado) [reply sidebars.js:299]: I don't believe so as of yet.

**Pull request closed without merging**